client/components/vertical-menu/items/social-item.jsx

```diff
@@ -0,0 +1,64 @@
+/** @ssr-ready **/
+
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import { localize } from 'i18n-calypso';
+import {
+	get,
+	identity
+} from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import SocialLogo from 'components/social-logo';
+
+const services = translate => ( {
+	facebook: { icon: 'facebook', label: translate( 'Facebook feed' ) },
+	google: { icon: 'google-plus-alt', label: translate( 'Google search' ) },
+	linkedin: { icon: 'linkedin', label: translate( 'LinkedIn share' ) },
+	twitter: { icon: 'twitter', label: translate( 'Twitter card' ) },
+	wordpress: { icon: 'wordpress', label: translate( 'WordPress.com reader' ) }
```

Comment:
@rralian I check with @deBhal before I did this in another PR before. If I am not mistaken, he told me that we're looking for _any_ function named `translate()` and that this would be fine in there.